FFXIVClientStructs/FFXIV/Client/Game/Object/GameObject.cs

```diff
@@ -203,6 +203,25 @@ public unsafe partial struct GameObject {
     /// </remarks>
     [MemberFunction("48 89 5C 24 ?? 48 89 6C 24 ?? 48 89 74 24 ?? 57 48 83 EC 20 48 8B 35 ?? ?? ?? ?? 48 8B F9")]
     public partial byte GetNamePlateColorType();
+
+    [MemberFunction("E8 ?? ?? ?? ?? 48 89 43 FB")]
+    public partial NamePlateColors GetNamePlateColorsRaw();
```

Comment:
```suggestion
    public partial NamePlateColors GetNamePlateColors();
```

Should match the name in `data.yml` as it is faster to lookup when setting the fields from the structs import